Sources/PaymentButtons/PaymentButtonSize.swift

```diff
@@ -16,16 +16,7 @@ public enum PaymentButtonSize: Int, CustomStringConvertible {
     case full
 
     var font: UIFont {
-        switch self {
-        case .mini, .collapsed:
-            return PaymentButtonFont.secondaryFont
-
-        case .expanded:
-            return PaymentButtonFont.primaryFont
-
-        case .full:
-            return PaymentButtonFont.systemFont18
-        }
+        return PaymentButtonFont.paypalPrimaryFont
```

Comment:
Can remove the keyword return and rely on implicit return here:
```suggestion
        PaymentButtonFont.paypalPrimaryFont
```